src/ExpressionEvaluator/Core/Source/FunctionResolver/FunctionResolver.cs

```diff
@@ -0,0 +1,189 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Debugger;
+using Microsoft.VisualStudio.Debugger.Clr;
+using Microsoft.VisualStudio.Debugger.ComponentInterfaces;
+using Microsoft.VisualStudio.Debugger.Evaluation;
+using Microsoft.VisualStudio.Debugger.FunctionResolution;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection.Metadata;
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    internal abstract class FunctionResolver :
+        FunctionResolverBase<DkmProcess, DkmClrModuleInstance, DkmRuntimeFunctionResolutionRequest>,
+        IDkmRuntimeFunctionResolver,
+        IDkmModuleInstanceLoadNotification
+    {
+        void IDkmRuntimeFunctionResolver.EnableResolution(DkmRuntimeFunctionResolutionRequest request, DkmWorkList workList)
+        {
+            if (request.LineOffset > 0)
```

Comment:
The old implementation calls request.OnResolverMessage per a previous comment I made.  Can you verify the behavior is acceptable without adding a resolution error message?